src/java/com/unifina/signalpath/blockchain/ContractEventPoller.java

```diff
@@ -31,10 +32,13 @@
 	private final String contractAddress;
 	private final EventsListener listener;
 	private String filterId;
-	private boolean keepPolling = false;
+	private boolean keepPolling = true;
```

Comment:
keepPolling starts true, and only turns false when close() is called

Reply:
The above comment would be useful as a comment in the code.